include/aos/common/tools/fs.hpp

```diff
@@ -24,6 +24,44 @@ namespace aos {
  */
 constexpr auto cFilePathLen = AOS_CONFIG_FS_FILE_PATH_LEN;
 
+/**
```

Comment:
We have fs namespace now instead of classes.  Move this interface to `spaceallocator.hpp`

Reply:
moved